submissions/krylenger/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,1342 @@
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat100">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat100"
+                  value="seat"
+                  aria-label="seat row"
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat101">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat101"
+                  value="seat"
+                  aria-label="seat row"
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat102">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat102"
+                  value="seat"
+                  aria-label="seat row"
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat103">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat103"
+                  value="seat"
+                  aria-label="seat row"
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat104">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat104"
+                  value="seat"
+                  aria-label="seat row"
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat105">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat105"
+                  value="seat"
+                  aria-label="seat row"
+                />
+                <span class="choose-section__span"></span>
+              </label>
+              <label class="choose-section__label" for="seat106">
+                <input
+                  class="choose-section__input visually-hidden"
+                  type="checkbox"
+                  name="seat"
+                  id="seat106"
+                  value="seat"
```

Comment:
[Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) about attribute `value`

Reply:
Yes, thanks. I've didn't add correct(unique) values because it was a little bit annoying for me to input all those unique values for each seat.
✅ Fixed!